vizro-core/examples/_dev/app.py

```diff
@@ -1,112 +1,89 @@
-                columnDefs=[
-                    {"field": "AAPL", "headerName": "Format Dollar", "cellDataType": "dollar"},
-                    {"field": "AAPL", "headerName": "Format Euro", "cellDataType": "euro"},
-                    {"field": "random", "headerName": "Format Numeric", "cellDataType": "numeric"},
-                    {"field": "perc_from_float", "headerName": "Format Percent", "cellDataType": "percent"},
-                    {
-                        "field": "perc_from_float",
-                        "headerName": "custom format",
-                        "valueFormatter": {"function": "d3.format('.^30')(params.value)"},
-                    },
-                ],
-                defaultColDef={"editable": True},
-                # dashGridOptions = {"pagination": False},
-            ),
+        vm.Card(
+            text="""
+              ### No Href
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
+              molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum
+              numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium
+              option, eaque rerum!
+          """,
+        ),
+        vm.Card(
+            text="""
+            ### /first-page
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
+            molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum
+            numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium
+            option, eaque rerum!
+        """,
+            href="/first-page",
+        ),
+        vm.Card(
+            text="""
+            ### First Page
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
+            molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum
+            numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium
+            option, eaque rerum!
+            """,
+            href="First Page",
+        ),
+        vm.Card(
+            text="""
+            ### https
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
+            molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum
+            numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium
+            option, eaque rerum!
+            """,
+            href="https://www.google.com",
+        ),
+        vm.Card(
+            text="""
+            ### http
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
+            molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum
+            numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium
+            option, eaque rerum!
+            """,
+            href="http://www.google.com",
+        ),
+        vm.Card(
```

Comment:
Is this one supposed to work, I get 404

Reply:
No, that's the expected result as you have it :) Just wanted to demo, that www.google.com is not considered an absolute path 👍